src/c++/perf_analyzer/genai-pa/genai_pa/main.py

```
@@ -65,8 +65,15 @@ def calculate_metrics(file: str) -> LLMProfileData:
     return LLMProfileData(file, t)
 
 
-def report_output(metrics: LLMProfileData):
-    print(metrics.get_statistics("concurrency", 1))
+def report_output(metrics: LLMProfileData, args):
+    if "concurrency_range" in args:
+        infer_mode = "concurrency"
+        load_level = args.concurrency_range
+    else:
+        infer_mode = "request_rate"
+        load_level = args.request_rate_range
```

nit: I would do `if` / `elif` / `else: raise GenAiException("unsupported mode")`